qa/rpc-tests/getblockexpanded.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal, initialize_chain_clean, \
+    start_node, connect_nodes_bi, assert_true, assert_false, get_epoch_data, \
+    swap_bytes, disconnect_nodes
+from test_framework.mc_test.mc_test import *
```

Comment:
It would be better to avoid importing unused symbols.

```suggestion
from test_framework.mc_test.mc_test import CertTestUtils, generate_random_field_element_hex
```